autorandr.py

```diff
@@ -496,7 +496,11 @@ def main(argv):
         print(str(e))
         options = { "--help": True }
 
-    profile_path = os.path.expanduser("~/.autorandr")
+    profile_dir = os.path.expanduser("~/.autorandr")
+    if not os.path.isdir(profile_dir):
+        profile_dir = os.path.join(os.environ.get("XDG_CONFIG_HOME", os.path.expanduser("~/.config")), "autorandr")
+
+    profile_path = os.path.join(profile_dir, "profiles")
```

Comment:
This line still makes the alternatives `~/.config/profiles` and `~/.autorandr/profiles`, which is wrong - the fallback should be just `~/.autorandr`. I'd suggest to just remove this line, i.e. to place profiles directly in `~/.config/autorandr`. `profiles` is misleading anyway, because the directory also holds the global `preswitch` and `postswitch` scripts.